src/SB/Game/zNPCFXCinematic.h

```diff
@@ -8,13 +8,68 @@
 #include "zNPCHazard.h"
 #include "zParEmitter.h"
 #include "zShrapnel.h"
+#include "rwcore.h"
+
+
+//#include "iColor.h"
```

Comment:
Please try to avoid including commented includes without some exceptional reason.